cmd/gossamer/config_test.go

```diff
@@ -1302,3 +1307,76 @@ func Test_setLogConfig(t *testing.T) {
 		})
 	}
 }
+
+// newTestGenesisRawFile returns a test genesis file using "gssmr" raw data
+func newTestGenesisRawFile(t *testing.T, cfg *dot.Config) (filename string) {
+	t.Helper()
+
+	filename = filepath.Join(t.TempDir(), "genesis.json")
+
+	fp, err := utils.GetGssmrGenesisRawPath()
+	require.NoError(t, err)
+
+	gssmrGen, err := genesis.NewGenesisFromJSONRaw(fp)
+	require.NoError(t, err)
+
+	gen := &genesis.Genesis{
+		Name:       cfg.Global.Name,
+		ID:         cfg.Global.ID,
+		Bootnodes:  cfg.Network.Bootnodes,
+		ProtocolID: cfg.Network.ProtocolID,
+		Genesis:    gssmrGen.GenesisFields(),
+	}
+
+	b, err := json.Marshal(gen)
+	require.NoError(t, err)
+
+	err = os.WriteFile(filename, b, os.ModePerm)
+	require.NoError(t, err)
+
+	return filename
+}
+
+// newTestGenesisAndRuntime create a new test runtime and a new test genesis
+// file with the test runtime stored in raw data and returns the genesis file
+func newTestGenesisAndRuntime(t *testing.T) (filename string) {
+	_ = wasmer.NewTestInstance(t, runtime.NODE_RUNTIME)
+	runtimeFilePath := runtime.GetAbsolutePath(runtime.NODE_RUNTIME_FP)
```

Comment:
to make it dumber / explicit

```suggestion
	runtimeFilePath := filepath.Join(t.TempDir(), "runtime")
	_, testRuntimeURL := runtime.GetRuntimeVars(runtime.NODE_RUNTIME)
	err := runtime.GetRuntimeBlob(runtimeFilePath, testRuntimeURL)
	require.NoError(t, err)
```

Reply:
Updated.